client/src/components/search-page/Search.js

```diff
@@ -19,7 +19,7 @@ export default function Search() {
       </div>
       <main className="main search-page">
         <SearchBar handleNameFilter={handleNameFilter} />
-        <Dogs filter={filter} />
+        <SlideUpTransition filter={filter} />
```

Comment:
I think is this a weird change, I would've expected something like this:
```suggestion
        <SlideUpTransition>
             <Dogs filter={filter} />
        </ SlideUpTransition>
```
OR add the transition to `Dogs`